### ISSUE TITLE
Fix resolver version or cargo

### ISSUE THREAD

**Issue by ClementWalter** [lambdaclass/cairo-vm#1260]
**Describe the bug**

**To Reproduce**

1. Pull the project on main branch
2. run `cargo build --workspace`
3. See the following warning

```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```

**Expected behavior**

A resolver version set up.

**Additional context**

See also https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions

**Comment by MegaRedHand** [comment]
Could you please share the output of `cargo --version` ?

**Comment by ClementWalter** [comment]
```
cargo --version
cargo 1.72.0-nightly (b0fa79679 2023-06-03)
```

**Comment by MegaRedHand** [comment]
Thanks for the heads-up! Should be fixed by #1311
